lib/screens/v2/widget_instance/departures.ex

```diff
@@ -68,13 +73,21 @@ defmodule Screens.V2.WidgetInstance.Departures do
 
     %{
       id: row_id,
+      type: :departure_row,
       route: serialize_route(departures),
       headsign: serialize_headsign(departures),
       times_with_crowding: serialize_times_with_crowding(departures, screen),
       inline_alerts: serialize_inline_alerts(departures)
     }
   end
 
+  defp serialize_row(%{text: %FreeTextLine{} = text}, _screen) do
```

Comment:
If you agree with the last suggestion, this will appear inside a list now.
```suggestion
  defp serialize_row([%{text: %FreeTextLine{} = text}], _screen) do
```